packages/core-components/src/components/HeaderIconLinkRow/IconLinkVertical.tsx

```diff
@@ -98,7 +96,7 @@ export function IconLinkVertical({
       title={title}
       className={classnames(classes.link, classes[color])}
       to={href}
-      onClick={onClick}
+      {...props}
```

Comment:
Hmm, are you actually looking to forward all props or just the `ref`? The `ref` won't be forwarded by this, you need to use [forwardRef](https://react.dev/reference/react/forwardRef) for that. Tbh I think it would be better to avoid forwarding all props here and keep the API for this component slim.